perllib/FixMyStreet/Cobrand/IsleOfWight.pm

```diff
@@ -86,6 +86,41 @@ sub munge_category_list {
     @$options = grep { my $c = ($_->{category} || $_->category); $c =~ 'Pick a category' || $seen->{ $c } } @$options;
 }
 
+sub munge_load_and_group_problems {
+    my ($self, $where, $filter) = @_;
+
+    return unless $where->{category};
+
+    my $b = $self->{c}->model('DB::Body')->for_areas( $self->council_area_id )->first;
+
+    my $cats = $self->{c}->model('DB::Contact')->search(
+        {
+            body_id => $b->id,
+            category => $where->{category}
+        }
+    );
+
+    my @cat_names;
+    while ( my $cat = $cats->next ) {
+        if ( $cat->send_method eq 'Triage' ) {
+            my $cat_name = $cat->category;
+            my $len = length $cat_name;
+            my @cats = $self->{c}->model('DB::Contact')->search(
```

Comment:
As there won't be many, but this loop might be a few times, wonder if it's easier/nicer code to fetch all the body's contacts first off, and create a lookup of group => categories, then you don't have to worry about RABX string lengths at least!